emissionsapi/download.py

```diff
@@ -35,26 +65,31 @@ def download():
     date_begin = config('download', 'date', 'begin')\
         or '2019-09-10T00:00:00.000Z'
     date_end = config('download', 'date', 'end') or '2019-09-11T00:00:00.000Z'
-    countries = config('download', 'country') or ['DE']
+    countries = config('download', 'country') or None
 
     # create storage folder if not existing
     os.makedirs(storage, exist_ok=True)
 
+    if countries is None:
+        logger.info(
+            f'Looking for products between {date_begin} and {date_end}.')
+        filtered_download(
+            polygon=None,
```

Comment:
Since you are using keyword arguments anyway, it's superfluous to state the default value here again.